stdlib/list.mli

```diff
@@ -332,42 +382,50 @@ val assoc : 'a -> ('a * 'b) list -> 'b
    [assoc a [ ...; (a,b); ...] = b]
    if [(a,b)] is the leftmost binding of [a] in list [l].
    @raise Not_found if there is no value associated with [a] in the
-   list [l]. *)
+   list [l].
+ *)
 
-val assoc_opt: 'a -> ('a * 'b) list -> 'b option
+val assoc_opt : 'a -> ('a * 'b) list -> 'b option
 (** [assoc_opt a l] returns the value associated with key [a] in the list of
-   pairs [l]. That is,
-   [assoc_opt a [ ...; (a,b); ...] = b]
-   if [(a,b)] is the leftmost binding of [a] in list [l].
-   Returns [None] if there is no value associated with [a] in the
-   list [l].
-   @since 4.05 *)
+    pairs [l]. That is,
+    [assoc a [ ...; (a,b); ...] = b]
```

Comment:
This is a description of `assoc`, not `assoc_opt`. Maybe something like:
```
 [assoc_opt a [ ...; (a,b); ...] = Some b]
```
?

Reply:
Fixed. Mistake was from trunk.